### ISSUE TITLE
Variable height or width

### ISSUE THREAD

**Issue by python-force** [SmileyChris/easy-thumbnails#181]
Hello,

Would like to ask I am trying to do this:

I know Imagefield in Django has PIL attributes so I can do:

{{ photo.image.width }} x {{ photo.image.height }} = size in from DB image PIL = 100 x 200 - example

Basically from the Imagefield getting the width and height.

When I am having this:

  img style="" src="{% thumbnail photo.image 160x500 %}" 

How can I get the information about width and height, I cant use above because that image has different dimensions lets say: 960 x 1400 and I am resizing to 160x500.

Is there a way get some thing like this:

  img width="{{thumbnail.width }}" height="{{ thumbnail.height }}"  src="{% thumbnail photo.image 160x500 %}"  /> but with proper sizing 160x"something"

The picture is resizing in propotions, 500 is the max. and I need something that I can get the picture height. Width is always 160 but when it resizes pictures have height: 239, 234, 265 etc...and I need that
value I cant predict it. How can I read dimesions of this picture and put it in the img width and height attributes

Basically how can I get precise dimensions of this thumbnail....
{% thumbnail photo.image 160x500 %}

I hope you understand....
